files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -718,3 +718,19 @@
     name: "nullmailer"
     state: "stopped"
   when: not ansible_is_chroot
+
+#
+# Add the correct path to /etc/security/pam_env.conf and remove any invalid
+# paths from /etc/environment to ensure that non-existent paths are not
+# included in the global PATH.
+#
+- lineinfile:
+    path: /etc/security/pam_env.conf
+    state: present
+    regexp: '^\s*PATH\s+DEFAULT='
+    line: 'PATH DEFAULT=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin'
+
+- lineinfile:
```

Comment:
This can be removed once the minimum upgrade version is 28.0.0.0. You can add a comment here to that effect (see the MIN_UPGRADE_VERSION description in `appliance/gradle.properties`.

Reply:
Yeah, it make sense, Add the comment.